packages/module/src/SkeletonTable/SkeletonTable.tsx

```diff
@@ -20,21 +22,43 @@ export type SkeletonTableProps = Omit<TableProps, 'ref'> & {
       }
   );
 
-
 function hasCustomColumns(props: Record<string, any>): props is SkeletonTableProps & {
   columns: ReactNode[];
 } {
   return Array.isArray(props.columns);
 }
 
 const SkeletonTable: React.FunctionComponent<SkeletonTableProps> = (props: SkeletonTableProps) => {
-  const { variant, rows = 5, caption, ouiaId = 'SkeletonTable', ...rest } = props;
+  const { variant, rows = 5, caption, ouiaId = 'SkeletonTable', isSelectable, isExpandable, sortBy, ...rest } = props;
   const rowCells = hasCustomColumns(props) ? props.columns.length : props.numberOfColumns;
   const rowArray = [ ...new Array(rowCells) ];
   const bodyRows = [ ...new Array(rows) ].map((_, rowIndex) => (
     <Tr key={rowIndex} ouiaId={`${ouiaId}-tr-${rowIndex}`}>
+      {
+        isExpandable && (
+          <Td
+            data-ouia-component-id={`${ouiaId}-td-expand-${rowIndex}`}
+            expand={{
+              rowIndex,
+              isExpanded: false
+            }}
+          />
+        )
+      }
+      { isSelectable && (
+        <Td
+          data-ouia-component-id={`${ouiaId}-td-select-${rowIndex}`}
+          select={{
+            rowIndex,
+            isSelected: false
```

Comment:
```suggestion
            isSelected: false,
            isDisabled: true
```
Let's make the checkboxes/radios disabled